browser/translate/brave_translate_browsertest.cc

```diff
@@ -258,7 +260,9 @@ IN_PROC_BROWSER_TEST_F(BraveTranslateBrowserTest, InternalTranslation) {
 
   SetupTestScriptExpectations();
 
-  auto* bubble = TranslateBubbleView::GetCurrentBubble();
+  auto* bubble = TranslateBubbleController::FromWebContents(
```

Comment:
@atuchin-m ptal

Reply:
Looks good.
All the tests check the bubble on the active WebContents, so the replacement should be legal.